llama_index/node_parser/text/sentence.py

```diff
@@ -170,7 +170,7 @@ def _split_text(self, text: str, chunk_size: int) -> List[str]:
         Has a preference for complete sentences, phrases, and minimal overlap.
         """
         if text == "":
-            return []
+            return [text]
 
```

Comment:
Let's change this in the TokenTextSplitter too

Reply:
you got it!